main.qml

```diff
@@ -1228,6 +1393,10 @@ ApplicationWindow {
         property int lockOnUserInActivityInterval: 10  // minutes
         property bool showPid: false
         property bool blackTheme: false
+        property bool fiatPriceEnabled: false
+        property bool fiatPriceEnabledCheckbox: false
```

Comment:
I don’t see a reason to save fiatPriceEnabledCheckbox here.

Reply:
That is correct, I've set it locally (`appWindow`) instead. Thanks.